examples/tutorials/Introduction to Langrangian Neural Networks.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.    #setting the lagrangian loss function (in the original paper, it was set to be the L2 distance between the original and predicted outcomes)
</pre><p>I belive Deepchem has L2 loss as a function, try to import and use that here instead of defining the loss function again</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3905/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>